packages/fxa-settings/src/components/Security/index.test.tsx

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import Security from '.';
+
+const { findByText, findAllByText } = screen;
+
+describe('SecurityPanel', async () => {
+  it('renders "fresh load" SecurityPanel with correct content', () => {
```

Comment:
We should probably update this from `SecurityPanel` to `Security` now.

Also when running these tests, I'm seeing this: "Warning: You seem to have overlapping act() calls, this is not supported. Be sure to await previous act() calls before making a new one." 🤔